doc/manual/citing_aspect.bib

```diff
@@ -2687,3 +2687,52 @@ @article{douglas2025constraining_solid_dynamics
 doi = {https://doi.org/10.1029/2024GC011824},
 year = {2025}
 }
+
+@phdthesis {2025frompoleto,
+	title = {From Pole to Shoreline: Constraining Mantle Structure and Flow to Understand Past Sea-Level Change},
+	publisher = {ProQuest Dissertations & Theses},
+	school = {Columbia University},
+	url = {https://www.proquest.com/dissertations-theses/pole-shoreline-constraining-mantle-structure-flow/docview/3198955800/se-2?accountid=14505},
+	volume = {31931588},
+	year = {2025},
+	author = {Hollyday, A. E.}
+}
+
+@phdthesis {2025investigat,
+	title = {Investigating the Kinematics and Dynamics of Strain Localization in the Northern Western Branch of the East African Rift System},
+        publisher = {Virginia Tech},
+	url = {https://hdl.handle.net/10919/134213},
+	year = {2025},
+	keywords = {ASPECT, Magma-poor continental rifting, GNSS geodesy, Kinematic modeling, Geodynamic modeling},
+	author = {Kwagalakwe, Asenath}
+}
+
+@Article{Gianni2025,
+author={Gianni, Guido M.
+and Guo, Zonglin
+and Holt, Adam F.
+and Faccenna, Claudio},
+title={Non-collisional orogeny in northeast Japan driven by nearby same-dip double subduction},
+journal={Nature Geoscience},
+year={2025},
+month={jun},
```

Comment:
I *think* that what biber is complaining about is that you can use "jun" as an abbreviation for "{6}" or some such, but that if you do, you can't surround it with braces because then it's interpreted as text that is not expanded. The field is defined as numeric, so non-number-text is not allowed.

Either way, I don't think it adds very much information, and so I would remove the `month` and `day` fields altogether.